brunch-config.js

```diff
@@ -52,6 +63,17 @@ exports.config = {
     babel: {
       // Do not use ES6 compiler in vendor code
       ignore: [/web\/static\/vendor/]
+    },
+    copycat: {
+      // copies to priv/static/fonts/
+      "fonts": ["node_modules/bootstrap-sass/assets/fonts/bootstrap"],
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)